src/engine/GameWorker.ts

```diff
@@ -139,10 +141,18 @@ async function onInit({
     return interval;
   };
 
-  gameLoop();
+  if (isFirefox) {
+    update(ctx);
+  } else {
+    gameLoop();
+  }
 }
 
 function update(ctx: GameState) {
+  if (isFirefox) {
+    setTimeout(() => update(ctx), 1000 / tickRate);
```

Comment:
Scheduling a `setTimeout` up front without taking into account the duration of the frame could cause issues if the frame overruns its tickrate.

Reply:
this unfortunately exacerbates the jittering :/

Reply:
Yeah 😕 bummer ok just needs a comment on why we use one or the other then